spec/beaker-hostgenerator/generator_spec.rb

```diff
@@ -98,12 +57,20 @@ module BeakerHostGenerator
             fixture_hash['environment_variables'].each do |key, value|
               ENV[key] = value
             end
+
             cli = BeakerHostGenerator::CLI.new(arguments)
-            test_hash = YAML.load(cli.execute)
+            if fixture_hash['expected_exception']
+              # Turn fully-qualified classname string into an actual Class object.
+              expected_class = eval fixture_hash['expected_exception']
```

Comment:
I'll change this to use my alternate implementation, and add a simple test fixture to exercise it.